cmd/clusterctl/cmd/describe_cluster.go

```diff
@@ -524,6 +530,63 @@ func getMultilinePrefix(currentPrefix string) string {
 	return "?"
 }
 
+// getRootMultiLinePrefix return the tree view prefix for a multiline condition on the object.
+func getMultilinePrefix(obj ctrlclient.Object, objectTree *tree.ObjectTree, prefix string) string {
+	// If it is the last object we can return early with an empty string.
+	orderedObjects := getOrderedTreeObjects(objectTree)
+	if orderedObjects[len(orderedObjects)-1].GetUID() == obj.GetUID() {
+		return ""
+	}
+
+	// Get the multiline prefix for the objects condition.
+	multilinePrefix := getMultilineConditionPrefix(prefix)
+
+	// If it is the top-level root object, set the multiline prefix to a pipe.
+	if prefix == "" {
+		multilinePrefix = pipe
+	}
+
+	// If the object is not the root object and has children, we need to add a pipe to the multiline prefix.
+	childrenObj := objectTree.GetObjectsByParent(obj.GetUID())
+	if obj.GetUID() != objectTree.GetRoot().GetUID() && len(childrenObj) > 0 {
+		// If the multiline prefix is empty, indent the multiline prefix first
+		if multilinePrefix == "" {
+			multilinePrefix = indent
+		}
+		multilinePrefix += pipe
+	}
+
+	// If the multiline prefix is empty, we have to ensure that even multiline conditions on this root object are indented.
+	if strings.TrimSpace(multilinePrefix) == "" {
+		filler := strings.Repeat(" ", 10)
+		childrenPipe := indent
+		if objectTree.IsObjectWithChild(obj.GetUID()) {
+			childrenPipe = pipe
+		}
+		// We just want to indent the multiline message for the Ready condition.
+		// All ├─ should be replaced by |, so all the existing hierarchic dependencies are carried on
+		multilinePrefix = strings.ReplaceAll(prefix+childrenPipe+filler, firstElemPrefix, pipe)
```

Comment:
I'm a little bit confused by having a filler at the end of the prefix given that what comes next goes in a different column (and so it automatically goes in the right position, no need to indent)
Same below for the additional spaces

Reply:
This is a copy from the code for the conditions

https://github.com/kubernetes-sigs/cluster-api/blob/eac73acbcd96923622edce57e5a38af3ee6ef17c/cmd/clusterctl/cmd/describe_cluster.go#L438-L439

This is necessary as the objects condition might have a multiline message and should be handled exactly as conditions via `--show-conditions`

Reply:
Maybe it makes sense to create a helper func `getObjectConditionPrefix()`, similar to `getChildPrefix()`. I can also just reference `getChildPrefix()` in a comment